cmd/version.go

```diff
@@ -41,7 +41,6 @@ DESCRIPTION
 	cmd.Run = func(cmd *cobra.Command, args []string) {
 		runVersion(cmd, args, version)
 	}
-	cmd.SetHelpFunc(defaultTemplatedHelp)
```

Comment:
Why is this line removed?

Reply:
Hm, this looks like a casualty of merge.  Thanks for noticing.

Reply:
I went through and checked all the other commands for a similar merge error. Should be GTG now.